index.html

```diff
@@ -306,31 +333,88 @@
 
 <h2 id="setup">Setup</h2>
 
-To participate in this workshop, you will need working copies of the 
-described software. Please make sure to install everything (or at least to download 
-the installers) before the start of your workshop. Participants should bring and use 
-their own laptops to insure the proper setup of tools for an efficient workflow 
+To participate in this workshop, you will need working copies of the
+described software. Please make sure to install everything (or at least to download
+the installers) before the start of your workshop. Participants should bring and use
+their own laptops to insure the proper setup of tools for an efficient workflow
 once you leave the workshop.
 
-<h3>Jupyter notebook</h3>
-
-The best way to install the Jupyter notebook is to use Anaconda. 
-<p>
-Regardless of how you choose to install it, please make sure you install Python version 3.x (e.g., 3.4 is fine).
+<h3>
+	Jupyter notebook
+</h3>
+The best way to install the Jupyter notebook is to use Anaconda.
 <p>
-
-<h3>Windows</h3>
-
-<ol>
-<li> Open <a href=http://continuum.io/downloads>http://continuum.io/downloads</a> with your web browser.
-<li> Download the Python 3 installer for Windows.
-<li> Install Python 3 using all of the defaults for installation except make sure to check *Make Anaconda the default Python*.
-</ol>
-
-<h3>Mac OSX</h3>
-
-<ol>
-<li> Open <a href=http://continuum.io/downloads>http://continuum.io/downloads</a> with your web browser.
-<li> Download the Python 3 installer for Mac OSX.
-<li> Install Python 3 using all of the defaults for installation 
-</ol>
+	Regardless of how you choose to install it, please make sure you install Python version 3.x (e.g., 3.4 is fine).
+	<p>
+		<div class="row">
+			<div class="col-md-6">
+				<h4>
+					Windows
+				</h3>
```

Comment:
No, they were `<h3>`s prior to my changes, as is **Setup**. So I demoted them to h4s but forgot to change the closing tags